### ISSUE TITLE
Does EmptyLineSeparator ignore custom tokens (8.36.2) ?

### ISSUE THREAD

**Issue by mchernyakov** [checkstyle/checkstyle#8975]
I would like to have `EmptyLineSeparator` without token `STATIC_INIT`.

```
/var/tmp $ javac Test.java
/var/tmp $ cat Test.java
package com.test;

public class Test {
    private static final int[] FIELD = new int[5];
    static {
        FIELD[0] = 16;
        FIELD[1] = 8;
        FIELD[2] = 6;
        FIELD[3] = 10;
        FIELD[4] = 10;
    }
}
```
I have configured `checkstyle.xml` with:
```bash
/var/tmp $ cat config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
    <module name="TreeWalker">
        <module name="EmptyLineSeparator">
            <property name="tokens"
            value="PACKAGE_DEF, IMPORT, STATIC_IMPORT, CLASS_DEF, INTERFACE_DEF, ENUM_DEF, INSTANCE_INIT,
            METHOD_DEF, CTOR_DEF, VARIABLE_DEF, RECORD_DEF, COMPACT_CTOR_DEF"/>
            <property name="allowNoEmptyLineBetweenFields" value="true"/>
            <property name="allowMultipleEmptyLines" value="false"/>
        </module>

    </module>

</module>
```
But still seeing that my build is failed, because of : 
```bash
/var/tmp $ java -jar checkstyle-8.36.2-all.jar -c config.xml Test.java
Starting audit...
[ERROR] /Users/mchernyakov/dev/tmp/Test.java:5:5:
   'STATIC_INIT' should be separated from previous statement. [EmptyLineSeparator]
Audit done.
Checkstyle ends with 1 errors.
```
Have I configured it normally? Or have I missed something? 
I am expecting no errors.

**Comment by romani** [comment]
Please report issue as it was required https://github.com/checkstyle/checkstyle/blob/master/.github/ISSUE_TEMPLATE/bug_report.md

**Comment by mchernyakov** [comment]
@romani updated, please check

**Comment by strkkk** [comment]
@mchernyakov you got response on stackoverflow
https://stackoverflow.com/questions/64856317/checkstyle-emptylineseparator-ignores-custom-tokens
